### PR TITLE
Add a video player with play/pause/fullscreen controls and progress bar.

### PR DESCRIPTION
**Pull request by Lateworm**
Add sample video content to projectResources page.

### CODE REVIEW

File: js/scenes/ProjectResources/ProjectResources.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import { Text, View } from "react-native";
-
 import { colors } from "../../config/styles";
 const { red } = colors;
+import VideoPlayer from "react-native-video-player";
```
Comment (siddharthparmar7) [inline js/scenes/ProjectResources/ProjectResources.js]: Please put imports before consts.

File: js/scenes/ProjectResources/ProjectResources.js [hunk 2]
```diff
@@ -13,6 +14,17 @@ const ProjectResources = () => (
     <Text>
       Carefully watch each video in sequence to prepare for your test.
     </Text>
+    <View style={styles.videoContainer}>
+      <VideoPlayer
+        // https://github.com/cornedor/react-native-video-player/blob/master/example/app/index.js
```
Comment (siddharthparmar7) [inline js/scenes/ProjectResources/ProjectResources.js]: Please remove this comment.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Thank you!

**Comment by Lateworm** [comment]
Hi Sid. The requested changes have been made. Thanks!

**Review by siddharthparmar7 — APPROVED** [review]
Thank you, Travis! Good work!

**Pull request merged**